egov-indexer/src/main/resources/application.properties

```diff
@@ -88,11 +88,13 @@ egov.pt.host=http://pt-services-v2:8080
 egov.pt.search.endpoint=/pt-services-v2/property/_search
 #..................................................................................#
 
-
+cache.expiry.mdms.masters.minutes=15
 
 # file path for loading yamls
 #egov.indexer.yml.repo.path=https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/watercharges-indexer.yml,https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/swm-service-indexer.yml,https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/asset-service-maha.yml,https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/lcms-indexer.yml,https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/inventory-service-indexer.yml,https://raw.githubusercontent.com/egovernments/egov-services/master/core/egov-indexer/src/main/resources/rainmaker-pgr-indexer.yml
-egov.indexer.yml.repo.path=file://home/vishal/git/master/egov-services/core/egov-indexer/src/main/resources/rainmaker-pt-indexer.yml
+egov.indexer.yml.repo.path=file:///Users/nithin/Documents/eGov/egov-repos/core-services/egov-indexer/src/main/resources/collection-indexer.yml
 
 
 logging.pattern.console=%clr(%X{CORRELATION_ID:-}) %clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}
+
+spring.main.allow-bean-definition-overriding: true
```

Comment:
Overriding bean should not be allowed, please fix the issue where bean is being overridden